<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_WelcomePage</class>
 <widget class="QDialog" name="Admin_WelcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>842</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ClosedHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Receptionist Welcome Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 223, 255);</string>
  </property>
  <widget class="QPushButton" name="WelcomePage_Change_Menu_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 232, 174);</string>
   </property>
   <property name="text">
    <string>Change a Menu Restaurant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Add_new_employee_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 255, 204);</string>
   </property>
   <property name="text">
    <string>Add a new employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_list_of_all_receptionist_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 134, 12);</string>
   </property>
   <property name="text">
    <string>List of All Receptionist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_List_cleaning_staff_PB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 151, 196);</string>
   </property>
   <property name="text">
    <string>List of all cleaning staff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Remove_Employee_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 154, 255);</string>
   </property>
   <property name="text">
    <string>Remove Employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomePage_header_L">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(201, 204, 255);</string>
   </property>
   <property name="text">
    <string>WELCOME TO ADMIN HOME PAGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="WelcomePage_Change_type_of_room_PB">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 212, 255);</string>
   </property>
   <property name="text">
    <string>Change the type of a room</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
